src/components/Checkbox/BaseCheckbox.js

```diff
@@ -26,14 +30,15 @@ const propTypes = {
     /** Additional styles to add to checkbox button */
     style: stylePropTypes,
 
-    /** Callback that is called when mousedown is triggered. */
-    onMouseDown: PropTypes.func,
-
     /** A ref to forward to the Pressable */
     forwardedRef: PropTypes.oneOfType([
         PropTypes.func,
         PropTypes.shape({current: PropTypes.instanceOf(React.Component)}),
     ]),
+
```

Comment:
```suggestion
/** A function that is called when the box/label is pressed */
```

Reply:
When these doc strings are added, the linter asks to add return types and params as well. So, should I add a non doc comment? Or add return types and param types too?

Reply:
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/9059945/204832804-4dd5d6f6-341e-4c31-a82b-c3897425b311.png">

Weird, i don't have any linter errors
Could you check that once more?